### ISSUE TITLE
Make it easier to spot problematic Mods

### ISSUE THREAD

**Issue by JasonKaisersmith** [CitiesSkylinesMods/AutoRepair#6]
This is a really great and useful mod.  But I would suggest to improve the output to make it more readable and easier to spot problems.

At the moment the compatibility statement is shown on the 3rd line of each Mod output and can be something like:

- Not compatible with current game version....
- Should be compatible with current game version .....
- Broken mod. Unsubscribe it.
- Confirmed compatible with Cities: Skylines v1.13.0 :)
etc.

The idea of this Mod is to highlight problems.  So it should be easy to find the problematic Mods, and at the moment the output makes it a little difficult to spot the issues.

I would suggest one of two things to improve this.

1)  Make the current output more obvious, with compatibility stated at the start of the line under each Mod name.  Starting with the compatibility statement itself.  Such as;
------
[ID: 812125426] "Network Extensions 2":
NOT COMPATIBLE with current game version....

 - [AutoRepair Descriptor] Catalog: 'Adden.....
.....
With a summarised statement such as "NOT COMPATIBLE", "Compatible", "Should be Compatible" etc.

2)   Keep the current output and also add a summary at the top or bottom of the output, maybe summarised per compatibility Statement. Such as 

Not Compatible Mods:
Network Extensions 2
Fine Road Anarchy
Automatic Bulldoze v2
..

Compatible Mods;
Enable Achievements
Precision Engineering
...

The idea of this Mod is to highlight problems.  So it should be easy to find the problematic Mods.

I can offer my services if you are interested.

**Comment by aubergine10** [comment]
Yup, I'm hoping to improve the logging later today. I have >100 mods installed so it's a massive wall of text for me too hehe.

**Comment by aubergine10** [comment]
I've added a settings screen with some options for what goes in to log file. By default there's now less info in log file. I've also rearranged some of the logging to put the compatibility stuff as close to top as possible.

I'll see if I can get a summary added to top of file later on.

**Comment by aubergine10** [comment]
Feedback from workshop page along similar lines:

> **theoobg** Just now   
> thanks aubergine for your quick feedback. I found a way to see the checked content below each other. Was just searching in the editor for "[ID:" and was moving it by pressing return to the next row. Now all checked content is listed down below. If you want to see how does it looks like, I can send it to you via chat if you want.  
> My status after the check:
>  
> Broken Mods are:  
> * "Building Search"
> * "Auto Line Color"
> * "Advanced Vehicle Options 1.8.2"
>  
> Not campatible Mods:  
> * "Network Extensions 2"
